Makefile

```diff
@@ -75,10 +78,11 @@ clean:
 	rm -r bin
 	rm -f $(SAMPLES)
 	rm -f config/rbac/role.yaml
+	rm -rf ./dist/*
 	find . -name "cover.out" -delete
 
 clean-deps:
```

Comment:
Why are we dropping the kubebuilder target here?

Reply:
Because it doesn't exist anymore: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/Makefile#L29-L38